onnx/test/optimizer_test.py

```diff
@@ -122,6 +122,21 @@ def check_identity(node):  # type: (NodeProto) -> None
         assert len(optimized_model.graph.node[2].attribute[0].g.output) == 2
         assert optimized_model.graph.node[2].attribute[0].g.output[1].name == "_Y"
 
+    def test_eliminate_identity_graph_output(self):  # type: () -> None
+        add = helper.make_node("Add", ["X", "Y"], ["A"])
+        identity = helper.make_node("Identity", ["A"], ["B"])
```

Comment:
This is a great test for `Identity`. May we have tests for `Dropout`, `Pad`, and `Transpose` as well? Thanks.

Reply:
Sure :) Thanks.

Reply:
Added